webtau-core-groovy/src/test/groovy/com/twosigma/webtau/data/table/TableDataExtensionTest.groovy

```diff
@@ -87,6 +96,14 @@ class TableDataExtensionTest {
          "v2a"                | permute(10, 20) | "v2c" }
     }
 
+    static TableData createTableDataWithPreviousRef() {
+        ["Col A" | "Col B" | "Col C"] {
+        __________________________________________
+            "v1a" |   "v1b" | 10
+            "v2a" |   "v2b" | cell.previous
```

Comment:
How about `cell.above`?  Or `row.previous`?  I'm slightly concerned that `cell.previous` makes it ambiguous whether it means previous row or column.

Reply:
I like `row.previous`. Wanted to have all the generator functions under one namespace. There is `cell.value(lambda)` and `cell.previous`, maybe more to come. but maybe it is ok.

Reply:
`cell.above` is growing on me.

Reply:
renamed